src/pages/Landing.jsx

```diff
@@ -33,6 +51,22 @@ export const Landing = () => {
         </Flex>
       </Box>
       <Link href="/wallet/123"> Draft of Wallet Page</Link>
+      <Box bg={bgMain} w="100%" p={4}>
+        <Flex bg={bgMain} direction="column" justify="center" align="center">
+          <Text fontSize="xl">
+            {i18next.t('transaction.recentTransactions')}
+          </Text>
+          <VStack w="80%" pt={5} spacing={5} align="stretch" justify="center">
+            {isFetched &&
```

Comment:
create a component which will have name TransactionsList and will accept array of transactions and do that logic inside